File: app/src/main/java/com/rendersoncs/reportform/view/adapter/listener/OnItemListenerClicked.java
package com.rendersoncs.reportform.view.adapter.listener;

public interface OnItemListenerClicked {
    void radioItemChecked(int itemPosition, int optNum);

    void takePhoto(int position);

    void fullPhoto(int position);

    void insertNote(int position);

    void updateList(int position);

    void removeItem(int position);

    void resetItem(int position);
}

File: app/src/main/java/com/rendersoncs/reportform/view/adapter/listener/OnItemListenerClicked.kt
package com.rendersoncs.reportform.view.adapter.listener

interface OnItemListenerClicked {
    /* Click choice radio button*/
    fun radioItemChecked(position: Int, optNum: Int)

    /* Click for take photo*/
    fun takePhoto(position: Int)

    /* Click for show details photo*/
    fun fullPhoto(position: Int)

    /* Click for insert note*/
    fun insertNote(position: Int)

    /* Click for update item list*/
    fun updateList(position: Int)

    /* Click for remove item list*/
    fun removeItem(position: Int)

    /* Click for reset list*/
    fun resetItem(position: Int)
}